x-pack/plugins/observability/README.md

```diff
@@ -0,0 +1,19 @@
+# Observability Shared Resources
+
+This "faux" plugin serves as a place to statically share resources, helpers, and components across observability plugins. There is some discussion still happening about the best way to do this, but this is one suggested method that will work for now and has the benefit of adopting our pre-defined build and compile tooling out of the box.
+
+Files found here can be imported from any other x-pack plugin, with the caveat that these shared components should all be exposed at the `public/index` and `server/index` (or `common/index` for code shared between client and server), so that the platform can attempt to monitor breaking changes in the shared API.
```

Comment:
> or `common/index` for code shared between client and server

This is not correct and the linter will soon reject imports that are not to `public/index` or `server/index`. If you have code that is applicable and safe on both, you will need to re-export that module in both places.